src/app/app-routing.module.ts

```diff
@@ -0,0 +1,47 @@
+import { AuthGuard } from './account/shared/auth.guard';
+import { CreateAccountComponent } from './account/create-account/create-account.component';
+import { LoginComponent } from './account/login/login.component';
+import { HomeComponent } from './layout/home/home.component';
+import { AppComponent } from './app.component';
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+import { AuthenticationComponent } from './layout/authentication/authentication.component';
+
+const routes: Routes = [
+    {
+        path: '',
+        component: HomeComponent,
+        children: [
+            {
+                path: '',
+                component: AppComponent
+            }
+        ],
+        canActivate: [AuthGuard]
```

Comment:
AuthGuard é responsável por verificar se pode ser acessado essa rota, se ele possui o token.
Toda alteração na rota ele entrar primeiro nele @Lucas-dOliveira

Reply:
show!

Reply:
Tudo o que for necessário estar logado, deverá ter esse canActivate: [AuthGuard]?

Reply:
Esse canActivate é um boolean que retorna se o usuário pode ou não acessar a rota interna do sistema.
Nesse caso ele valida o token ali dentro